config/environments/development.rb

```diff
@@ -9,14 +9,14 @@
   # Do not eager load code on boot.
   config.eager_load = false
 
-  # config.paperclip_defaults = {
-  #     :storage => :s3,
-  #     :s3_credentials => {
-  #         :bucket => ENV['S3_BUCKET_NAME'],
-  #         :access_key_id => ENV['AWS_ACCESS_KEY_ID'],
-  #         :secret_access_key => ENV['AWS_SECRET_ACCESS_KEY']
-  #     }
-  # }
+config.paperclip_defaults = {
+     :storage => :s3,
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.